docs/deployment/configuration/auth_setup.rst

```diff
@@ -17,13 +17,16 @@ The following diagram summarizes the components and their interactions as part o
 
 In summary, there are two main resources required for a complete auth flow in Flyte:
 
-**An identity layer**
+**Identity Layer**
 
-Using an implementation of the `Open ID Connect (OIDC) specification <https://openid.net/specs/openid-connect-core-1_0.html>`__, it enables clients to verify the identity of the end user based on the authentication performed by an Authorization Server. For this flow to work, you must first register Flyte as a new client (app) to the Identity Provider (IdP).
+The `OpenID Connect (OIDC) protocol <https://openid.net/specs/openid-connect-core-1_0.html>`__ allows clients (such as Flyte) to confirm the identity of a user, based on authentication done by an Authorization Server. To enable this, you first need to register Flyte as an app (client) with your chosen Identity Provider (IdP).
 
 **An authorization server**
 
-As defined by IETF's `RFC #6749 <https://datatracker.ietf.org/doc/html/rfc6749>`__, the authorization server's role is to issue *access tokens to the client after successfully authenticating the resource owner and obtaining authorization*. In this context, the *resource owner* is the end user of Flyte; and the *client* is the tool or component that intends to interact with ``flyteadmin`` : ``flytepropeller``, ``flyteconsole`` or any of the CLI tools.
+As defined by IETF's `RFC #6749 <https://datatracker.ietf.org/doc/html/rfc6749>`__ , the authorization server’s job is to issue access tokens to clients after verifying the user (resource owner) and getting their consent. In this setup:
+
+   1. The user of Flyte, is the *resource owner*.
```

Comment:
Not sure if we'll have to remove indentation here unless the intent for this to render like:

![image](https://github.com/user-attachments/assets/c79bb9c0-46f8-4a46-a9fb-c7dfffdc3fa0)

Reply:
I tried it without indentation but then the whole authorization server paragraph becomes italics for some reason. If you can suggest a fix for that please do.